assignments/callbacks.js

```diff
@@ -21,32 +21,74 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 */
 
-
+ // getLength passes the length of the array into the callback.
 function getLength(arr, cb) {
-  // getLength passes the length of the array into the callback.
+  return cb(arr.length);
 }
 
+getLength(items, function(length) {
+  console.log(length);
+});
+
+
+// last passes the last item of the array into the callback.
 function last(arr, cb) {
-  // last passes the last item of the array into the callback.
+  return cb(arr[arr.length -1]);
 }
 
+last(items, function(lastItem) {
+  console.log(lastItem);
+});
+
+ // sumNums adds two numbers (x, y) and passes the result to the callback.
 function sumNums(x, y, cb) {
-  // sumNums adds two numbers (x, y) and passes the result to the callback.
+  return cb(x + y);
 }
 
+const add = function(x) {
+  console.log(x);
+}
+
+sumNums(5, 5, add);
+
+// multiplyNums multiplies two numbers and passes the result to the callback.
 function multiplyNums(x, y, cb) {
-  // multiplyNums multiplies two numbers and passes the result to the callback.
+  return cb(x * y);
+}
+
+const multiply = function(x) {
+  console.log(x);
 }
 
+multiplyNums(5, 5, multiply);
+
+// contains checks if an item is present inside of the given array/list.
+// Pass true to the callback if it is, otherwise pass false.
 function contains(item, list, cb) {
-  // contains checks if an item is present inside of the given array/list.
-  // Pass true to the callback if it is, otherwise pass false.
+  if (list.includes(item)) {
```

Comment:
`list.includes` is a method which returns a boolean value. Knowing this we could shorten this code by simply writing `cb(list.includes(item))`;